### DISCUSSION TITLE
JDBC ResultSet 返回结果被截断

### DISCUSSION THREAD

**Discussion by imperio-wxm** [apache/kyuubi#3985] [Q&A]
结果集是一个比较长的MD5字符串，但是通过ResultSet获取后被截断，支持最大长度是多少？是否可以修改？
```java
 try (ResultSet rs = stmt.executeQuery(sql)) {
      while (rs.next()) {
          String text = rs.getString(1);
          System.out.println(text);
      }
  }
```
![image](https://user-images.githubusercontent.com/8111349/207815762-d56befec-b197-47c5-ad9f-d42c555002cb.png)

**Comment by yaooqinn** [comment]
原始长度是多少，返回的是多少？

**Reply by imperio-wxm** [reply]
> 原始长度是多少，返回的是多少？

原始5495个字符，返回785个字符。通过scala code方式执行

**Reply by yaooqinn** [reply]
string 应该只有2GB的物理限制，如果截断可能是bug
